liana-gui/src/app/wallet.rs

```diff
@@ -102,48 +102,17 @@ impl Wallet {
     }
 
     pub fn load_from_settings(self, dir: &NetworkDirectory) -> Result<Self, WalletError> {
-        let wallet = match settings::Settings::from_file(dir) {
-            Ok(settings) => {
-                if let Some(wallet_setting) = settings.wallets.first() {
-                    self.with_name(wallet_setting.name.clone())
-                        .with_hardware_wallets(wallet_setting.hardware_wallets.clone())
-                        .with_key_aliases(wallet_setting.keys_aliases())
-                        .with_provider_keys(wallet_setting.provider_keys())
-                } else {
-                    self
-                }
-            }
-            Err(settings::SettingsError::NotFound) => {
-                let s = settings::Settings {
```

Comment:
We do not create a new file anymore in this method. The file will be created in case of `update_settings_file` is called and the file does not exist anymore